src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/StepEntityMapperTest.kt

```diff
@@ -73,4 +75,44 @@ class StepEntityMapperTest {
     // Then
     assertThat(actual).usingRecursiveComparison().isEqualTo(expected)
   }
+
+  @Test
+  fun `should update entity from domain without updating JPA managed fields`() {
```

Comment:
Do we need this test and associated functionality?

Reply:
Good spot - I'll check (but you are likely right!)